src/components/Editable.tsx

```diff
@@ -525,6 +526,42 @@ const Editable = ({
     [value, setCursorOnThought],
   )
 
+  /**
+   * Handles void area taps by preventing default and setting the caret.
+   * Returns true if we handled the tap (void area), false if browser should handle it (valid tap on character).
+   */
+  const handleVoidAreaTap = useCallback(
+    (clientX: number, clientY: number, preventDefault: () => void): boolean => {
```

Comment:
I don't understand why `preventDefault` is being structured as a callback here. It's redundant with the return type, yet written as if it is independent. It should be handled by the caller.